frontend/components/UserSettings.vue

```diff
@@ -99,7 +99,15 @@ function changeColorMode() {
     } else if (selectedColorMode.value.name == "Lightmode") {
         colorMode.preference = "light";
     } else {
-        colorMode.preference = "System";
+        colorMode.preference = "system";
+    }
+}
+
+function checkUsername() {
+    if (username.value != "") {
+        onSubmitUsername();
```

Comment:
is there a reason why this method is named **on**SubmitUsername and not submitUsername?

Reply:
onSubmit isn't the actual submit function. In onSubmit the validation method handleSubmit from vee-validate validates the values and after that changeUsername() is called where the data is submitted

Reply:
I'd still say that the naming scheme with "on" would be more fit for something like an effect but since in this case the method acts in a similar way to an effect, the naming is fine